entrypoint.sh

```diff
@@ -10,6 +10,8 @@ if [ ! -e $image_path ]; then
       echo "Extracting fresh filesystem..."
       unzip $zip_path
       mv -- *.img $image_path
+
+      qemu-img resize $image_path 2G
```

Comment:
What is the reasoning behind this? Just to increase the filesystem size for normal use? Or is it required for QEMU 5.1?

If not required, it should be a separate PR.

Reply:
This is required for 64 bit based CPUs and is required for all at qemu 5.2

Reply:
Thanks!

So there's just a hard limit that all images must be no less that 2GB in size? Do you have source you could point me to so I can read more about that?

Reply:
It's been so long I can't remember if it's in the docs but I know it fails with an error message if you try with anything that is not a multiple of 2g

Reply:
Found it: mail-archive.com/qemu-devel@nongnu.org/msg723028.html